app/assets/javascripts/discourse/tests/integration/components/user-menu/menu-test.js

```diff
@@ -0,0 +1,55 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { query, queryAll } from "discourse/tests/helpers/qunit-helpers";
+import { render } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+
+module("Integration | Component | user-menu", function (hooks) {
+  setupRenderingTest(hooks);
+
+  const template = hbs`<UserMenu::Menu/>`;
+
+  test("notifications panel has a11y attributes", async function (assert) {
+    await render(template);
+    const panel = query("#quick-access-all-notifications");
+    assert.strictEqual(panel.getAttribute("tabindex"), "-1");
+    assert.strictEqual(
+      panel.getAttribute("aria-labelledby"),
+      "user-menu-button-all-notifications"
+    );
+  });
+
+  test("active tab has a11y attributes that indicate it's active", async function (assert) {
+    await render(template);
+    const activeTab = query(".top-tabs.tabs-list .btn.active");
+    assert.strictEqual(activeTab.getAttribute("tabindex"), "0");
+    assert.strictEqual(activeTab.getAttribute("aria-selected"), "true");
+  });
+
+  test("core top tabs", async function (assert) {
```

Comment:
`core` is a little vague here so I'm wondering if we can make this description more descriptive.